### PR TITLE
add chainIDs

### PR DESCRIPTION
**Pull request by finleydecker**

### CODE REVIEW

File: selectors.yml [hunk 1]
```diff
@@ -33,10 +39,13 @@ selectors:
   2358:
     selector: 5990477251245693094
     name: "ethereum-testnet-sepolia-kroma-1"
-  4002: 
+  2522:
+    selector: 8901520481741771655
+    name: "fraxtal-testnet-optimism-1"
```
Comment (andrevmatos) [inline selectors.yml:44]: This name indicates this is the first instance of a `optimism` (like, official) rollup being deployed to a unnamed/first/main testnet for the `fraxtal` L1. I was not aware of Optimistic Labs deploying official instances to these other L1s? Can you please provide more information to this network, and make sure it complies with the [naming pattern](https://www.notion.so/chainlink/Network-naming-proposal-161255913454455c9ba01f5cc25e5379)?

File: selectors.yml [hunk 2]
```diff
@@ -1,17 +1,23 @@
...
     selector: 13264668187771770619
     name: "binance_smart_chain-testnet"
   280:
     selector: 6802309497652714138
     name: "ethereum-testnet-goerli-zksync-1"
-  300: 
-    selector: 6898391096552792247 
+  296:
+    selector: 222782988166878823
+    name: "hedera-testnet"
```
Comment (andrevmatos) [inline selectors.yml:14]: Same: is this a true L1? And main/only/unnamed testnet?

Comment (finleydecker) [reply selectors.yml:14]: I believe it is a true L1. Here are some links if you could help me confirm.

And yes the testnet is unnamed. They just call it "Hedera Testnet".

[Hedera Website](https://hedera.com/learning/hedera-hashgraph/what-is-hedera-hashgraph)
[Hedera Docs](https://docs.hedera.com/hedera/#set-up-your-environment)
[Hedera Questionnaire](https://docs.google.com/spreadsheets/d/1lTYmYTY8SdFRsU35LOeZqVdDZipI3j06HbQ1BT0QN9c/edit#gid=625078687)

File: selectors.yml (hunk 1 above)
Comment (andrevmatos) [reply selectors.yml:44]: From what I can see from their site, they're a L2, anchored to Ethereum as its L1. Optimism is just the technology.
Their testnet is attached to Ethereum's [Holesky](https://holesky.fraxscan.com/) testnet.
Therefore, the naming should be:
```suggestion
    name: "ethereum-testnet-holesky-fraxtal-1"
```

File: selectors.yml (hunk 2 above)
Comment (andrevmatos) [reply selectors.yml:14]: Yes, looks like a true L1. Should be good to go. Thanks

File: selectors.yml (hunk 1 above)
Comment (finleydecker) [reply selectors.yml]: Got it thank you! Updated.

File: generated_chains.go [hunk 3]
```diff
@@ -43,8 +44,10 @@ var (
 	ETHEREUM_TESTNET_SEPOLIA_SCROLL_1       = Chain{EvmChainID: 534351, Selector: 2279865765895943307, Name: "ethereum-testnet-sepolia-scroll-1"}
 	ETHEREUM_TESTNET_SEPOLIA_ZKSYNC_1       = Chain{EvmChainID: 300, Selector: 6898391096552792247, Name: "ethereum-testnet-sepolia-zksync-1"}
 	FANTOM_TESTNET                          = Chain{EvmChainID: 4002, Selector: 4905564228793744293, Name: "fantom-testnet"}
+	FRAXTAL_TESTNET_OPTIMISM_1              = Chain{EvmChainID: 2522, Selector: 8901520481741771655, Name: "fraxtal-testnet-optimism-1"}
```
Comment (andrevmatos) [inline generated_chains.go]: Please, generate chains again

Comment (finleydecker) [reply generated_chains.go]: done

**Review by andrevmatos — APPROVED** [review]
Thank you

**Pull request merged**